src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -0,0 +1,135 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useEffect, useState, useRef } from 'react';
+import { Device, AudioTransformDevice, VoiceFocusTransformDevice } from 'amazon-chime-sdk-js';
+
+import { ControlBarButton } from '../../ui/ControlBar/ControlBarButton';
+import { DeviceConfig } from '../../../types';
+import { isOptionActive, audioInputSelectionToDevice } from '../../../utils/device-utils';
+import { Microphone } from '../../ui/icons';
+import { PopOverItemProps } from '../../ui/PopOver/PopOverItem';
+import { PopOverSeparator } from '../../..';
+import { Spinner } from '../../ui/icons';
+import { useMeetingManager } from '../../../providers/MeetingProvider';
+import { useAudioInputs } from '../../../providers/DevicesProvider';
+import { useVoiceFocus } from '../../../providers/VoiceFocusProvider';
+import { useToggleLocalMute } from '../../../hooks/sdk/useToggleLocalMute';
+
+interface Props {
+  /** The label that will be shown when microphone is muted, it defaults to `Mute`. */
+  muteLabel?: string;
+  /** The label that will be shown when microphone is unmuted, it defaults to `Unmute`. */
+  unmuteLabel?: string;
+  /** Title attribute for the icon when muted, it defaults to `Muted microphone` */
+  mutedIconTitle?: string;
+  /** Title attribute for the icon when unmuted, it defaults to `Microphone` */
+  unmutedIconTitle?: string;
+}
+
+const AudioInputVFControl: React.FC<Props> = ({
+  muteLabel = 'Mute',
+  unmuteLabel = 'Unmute',
+  mutedIconTitle,
+  unmutedIconTitle
+}) => {
+  const meetingManager = useMeetingManager();
+  const [voiceFocusOn, setVoiceFocusOn] = useState(false);
```

Comment:
`voiceFocusOn` sorry do not see where the state for this is changed, could you please let me know if this is needed?

Reply:
Sure, I changed this `voiceFocusOn` to `isVoiceFocusOn` for better understanding. And this `isVoiceFocusOn` is updated when the user clicks the voice focus button.